api/src/main/java/com/datastrato/gravitino/rel/SchemaChange.java

```diff
@@ -47,6 +60,22 @@ static SchemaChange removeProperty(String property) {
     return new RemoveProperty(property);
   }
 
+  /** A SchemaChange to update a schema's comment. */
+  @EqualsAndHashCode
+  @Getter
+  final class UpdateComment implements SchemaChange {
+    private final String newComment;
```

Comment:
Maybe it's better to use `comment` for variable names?

Reply:
@xunliu, I refers to variable name of `TableChange.UpdateComment` which also uses `newComment ` as variable name.